src/Models/SingleColumnModels/equations_base_model.jl

```diff
@@ -0,0 +1,116 @@
+@inline function rhs_base_model!(dY, Y, Ya, t, _...)
+    error("not implemented for this model configuration.")
+end
+
+function rhs_base_model!(
+    dY,
+    Y,
+    Ya,
+    t,
+    # Π, # just for now, incremental tests -> climacore ekman, p,
+    p,
+    Φ,
+    ::AdvectiveForm,
+    params,
+    flux_correction,
+    FT,
+)
+    # base components
+    dρ = dY.base.ρ # scalar on centers
+    duh = dY.base.uh # Covariant12Vector on centers
+    dw = dY.base.w # Covariant3Vector on faces
+    ρ = Y.base.ρ
+    uh = Y.base.uh
+    w = Y.base.w
+    ρθ = Y.thermodynamics.ρθ
+
+    # experiment specif parameters
+    # TODO: given lat in params, f = @. Geometry.Contravariant3Vector(Geometry.WVector(2*Ω*params.lat,)) 
+    ν = params.ν
+    Cd = params.Cd
+    f = params.f
+    uh_g = params.uh_g
+
+    # # gather boundary conditions:
+    # TODO: need to unpack outside rhs function and only pass in base bc
+    # bc_ρ = model.boundary_conditions.ρ
+    # bc_uh = model.boundary_conditions.uh
+    # bc_w = model.boundary_conditions.w
+
+    # density
+    # bc as hard coded for now; will later on make it work with the bc module
+    # flux_bottom = get_boundary_flux(model, bc_ρ.bottom, ρ, Y, Ya)
+    # flux_top = get_boundary_flux(model, bc_ρ.top, ρ, Y, Ya)
+    flux_bottom = Geometry.WVector(FT(0))
+    flux_top = Geometry.WVector(FT(0))
+    interp_c2f = Operators.InterpolateC2F()
+    vector_vdiv_f2c = Operators.DivergenceF2C(
+        top = Operators.SetValue(flux_top),
+        bottom = Operators.SetValue(flux_bottom),
+    )
+    @. dρ = -vector_vdiv_f2c(interp_c2f(ρ) * w)
```

Comment:
This is a compressible model. How would you suggest setting up an anelastic model? maybe we should separate the `ρ` equation from out of the `equations_base_model.jl` to `equations_density_model.jl` and dispatch the call to compressible and anelastic?

Reply:
I suggest an `Anelastic` option for `base` because we will also remove `w` equation there.

Reply:
Let's add the Anelastic Option in a separate PR